### PR TITLE
Update of DNS servers

### PR DESCRIPTION
**Pull request by rpeter85**
Added DNS servers and their description.
The following requests can be closed:
blokadaorg/blokada#241
blokadaorg/blokada#383

And Alternate DNS from this request:
blokadaorg/blokada#365

### CODE REVIEW

File: api/v3/canonical/defaults/dns.txt [hunk 1]
```diff
@@ -102,4 +137,3 @@ inactive
 ipv4
 77.88.8.7;77.88.8.3
 https://dns.yandex.com/advanced/
-
```
Comment (kar) [inline api/v3/canonical/defaults/dns.txt]: this empty line has to be here

File: api/v3/canonical/defaults/filters.txt [hunk 2]
```diff
@@ -128,7 +128,6 @@ https://zerodot1.gitlab.io/CoinBlockerLists/hosts
 b_app_vending
 whitelist
 active
-http://blokada.org
```
Comment (kar) [inline api/v3/canonical/defaults/filters.txt]: url addresses have to be here

File: api/v3/canonical/defaults/filters.txt [hunk 3]
```diff
@@ -245,8 +232,21 @@ com.waze
...
 app
 com.xda.labs
 
 
+130
+b_app_incallui
+whitelist
+active
```
Comment (kar) [inline api/v3/canonical/defaults/filters.txt]: need url address after this line (next entry also)

**Comment by rpeter85** [comment]
Requested changes are done.

**Pull request merged**